src/test/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletControllerTest.java

```diff
@@ -31,6 +39,23 @@
                 WalletModelMapper.class}))
 class WalletControllerTest {
 
+    private static final Long ID_OF_WALLET_1 = 1L;
```

Comment:
I believe you have suggested somebody else's code and made a whole list of constants used in the test methods below. Let's check ...

Reply:
I suggested the naming convention for the constants created in the previous PR